app/models/vm_reconfigure_request.rb

```diff
@@ -34,10 +34,9 @@ def self.request_limits(options)
     result[:max__cores_per_socket] = all_cores_per_socket.min
     result[:max__total_vcpus] = all_total_vcpus.min
 
-    result[:max__number_of_sockets] = 1 if result[:max__number_of_sockets].nil?
-    result[:max__cores_per_socket] = 1 if result[:max__cores_per_socket].nil?
+    result[:max__number_of_sockets] ||= 1
+    result[:max__cores_per_socket] ||= 1
     result[:max__vm_memory] ||= default_max_vm_memory
-    result[:max__total_vcpus] = 1 if result[:max__number_of_sockets].nil? && result[:max__cores_per_socket].nil?
```

Comment:
Nice find.  :scissors: 

Right, what was the original intention of that conditional?  Is there a value of `all_total_vcpus.min` that is invalid in which we'd want `result[:max__total_vcpus] = 1`?